iquip/apps/scheduler.py

```
@@ -73,7 +78,7 @@ def run(self):
                             arguments=expid["arguments"]
                         ))
                     self.fetched.emit(schedule)
-        except WebSocketException:
+        except:  # pylint: disable=bare-except
```

AFAIK, the bare except is in fact not desirable. Instead, `except Exception` would be better.
The bare except will prevent system level halts as well, for example Keyboard interrupt.

---

Oh, I didn't know much about the bare except.
I will apply your suggestion.

---

Umm.. Your solution causes a [broad-exception-caught](https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/broad-exception-caught.html).
Is it okay to disable this warning?

---

Yes, since this is for preventing the program not to crash and we are logging the exception properly, we can catch such a broad exception.